simplerunner.lua

```diff
@@ -63,53 +109,276 @@ function runbench(name, count)
     if not run_iter then
         error("Missing benchmark function for '"..name.."'")
     end
+    
+    if loading_jitstats and jitstats.stats.starts > 0 then 
+        jitstats.print("Traces created while loading benchmark")
+        jitstats.reset()
+    end  
+end  
 
-    count = count or 1
+function runbench(name, count, scaling)
+    scaling = scaling or 1
+    loadbench(name)

```

Comment:
Can we unload the benchmark at the end of the benchmarking?

Reply:
well i do nil out the run_iter global function but that doesn't really do much unless we run the garbage collector

Reply:
I think it would be fine to run the garbage collector after an unload. Running multiple benchmarks in the same process has weird effects; anything we can do to lessen those effects is a good idea (even though I know we can't do *that* much).